spec/features/user/users_profile_spec.rb

```diff
@@ -27,5 +27,25 @@
         expect(page).to have_content(donation.amount)
       end
     end
+
+    it 'can cancel an active subscription' do
+      expect(subscription.active).to eq(true)
+      visit user_path(user)
+      expect(page).to have_content(user.name)
+      expect(page).to have_content(user.email)
+
+      expect(page).to have_content(subscription.plan.name)
+
+      click_button 'Cancel Subscription'
+
+      expect(page).to have_content('Do you want to terminate your current subscription?')
+      within '#cancel-subscription-dialog' do
+        click_button 'Cancel Subscription'
+      end
+
+      sleep 2
```

Comment:
@castrolem this will probably lead to flaky tests, can we find a better approach?